meta/runtime.yml

```diff
@@ -255,46 +257,50 @@ plugin_routing:
       deprecation:
         removal_version: 2.0.0
         warning_text: see plugin documentation for details
+    gc_storage:
+      redirect: community.google.gc_storage
     gcdns_record:
-      deprecation:
-        removal_version: 2.0.0
-        warning_text: see plugin documentation for details
+      redirect: google.cloud.gcp_dns_resource_record_set
```

Comment:
I think the deprecations need to be tombstones, not redirects. The alternatives are not compatible replacements, but different modules which can be used to achieve the same goals.

Reply:
@felixfontein Thanks for the feedback -- it didn't feel right to put these redirects but I did the exercise of finding out what were the "replacements" from the module plugin docs, hence first bullet point in TODO :)

Some of these modules were moved to community.google ([and are deprecated there](https://github.com/ansible-collections/community.google/blob/main/meta/runtime.yml)) so should we redirect to community.google and let community.google handle the deprecation/tombstone ?

@gundalow do you have an opinion ?

Reply:
You can update the tombstone warning_text to contain information on the replacement. But adding a redirect is wrong.

I wouldn't add redirects to another tombstone, I'd simply tombstone them here. community.google shouldn't even have them in a first release.

Reply:
Yup, I agree with Felix's comment above